src/js/components/Peak/PeakNavBar.tsx

```diff
@@ -0,0 +1,60 @@
+import cx from 'classnames';
+
+import styles from './PeakNavBar.module.css';
+import { Link, useLocation } from 'react-router-dom';
+import React from 'react';
+
+type statesType = {
+  [name: string]: number;
+};
+
+const states: statesType = {
+  description: 1,
+  map: 2,
+  posts: 3,
+};
+
+const PeakNavBar = ({ id }: { id: number }) => {
+  const location = useLocation();
+
+  const getNavBarState = () => {
+    const params = location.pathname.split('/');
+    return states[params[3]] ?? states.description;
+  };
+
+  return (
+    <div className={styles.navigation}>
```

Comment:
W sumie to zastanawiam się czy powinno to być zrealizowane za pomocą osobnych URL, czy nie. Można się nad tym zastanowić później czy różne URL nie są tutaj przekombinowane

Reply:
Zmieniłam już w nowym PR